static/js/todo_widget.js

```diff
@@ -197,6 +228,16 @@ export function activate(opts) {
         const widget_data = task_data.get_widget_data();
         const html = render_widgets_todo_widget_tasks(widget_data);
         $elem.find("ul.todo-widget").html(html);
+        Sortable.create($elem.find("ul.todo-widget")[0], {
+            onUpdate(e) {
+                const old_pos = e.oldDraggableIndex;
+                const new_pos = e.newDraggableIndex;
+                const old_idx = (old_pos + 1) * 2;
```

Comment:
Everything looks and works alright! 

I just didn't understand why are we multiplying `old_pos` and `new_pos` with 2.

Reply:
the way `idx` is assigned initially makes it so that it starts at 2 for a task list and increments by 2 for each task, so for three tasks their `idx` values would be 2, 4, and 6, while the actual indices in the tasks array would of course be 0, 1 and 2. Hence the addition by 1 and doubling to convert from array index to `idx`

Reply:
This is unnecessary complexity to the code. Can we keep the `idx` values and indices same?